### ISSUE TITLE
Bug: Gluetun has started lowercasing my environment variable for Country, and thus not accepting it.

### ISSUE THREAD

**Issue by bjornsivertsen** [qdm12/gluetun#1600]
### Is this urgent?

None

### Host OS

Debian Bullseye

### CPU arch

None

### VPN service provider

Mullvad

### What are you using to run the container

docker run

### What is the version of Gluetun

Running version latest built on 2023-05-25T12:12:44.932Z (commit a43973c)

### What's the problem 🤔

Suddenly, Gluetun has started interpreting my environment variable in lowercase, albeit it isn't, and thus fails to make a connection.

Changed happened after recent release.

Snippit from my docker compose file:

```
    environment:
      - VPNSP=mullvad
      - VPN_TYPE=wireguard
      - WIREGUARD_PRIVATE_KEY=[redacted]
      - WIREGUARD_ADDRESS=[redacted]
      - SERVER_CITIES=Copenhagen
```

If I comment out the variable, it works.

### Share your logs

```plain text
2023-05-25T15:02:29+02:00 ERROR VPN settings: provider settings: server selection: for VPN service provider mullvad: the city specified is not valid: value is not one of the possible choices: value copenhagen, choices available are Adelaide, Amsterdam, Ashburn VA, Athens, Atlanta GA, Auckland, Belgrade, Bogota, Bratislava, Brisbane, Brussels, Bucharest, Budapest, Chicago IL, Chisinau, Copenhagen, Dallas TX, Denver CO, Dubai, Dublin, Dusseldorf, Frankfurt, Gothenburg, Helsinki, Hong Kong, Houston TX, Johannesburg, Lisbon, London, Los Angeles CA, Luxembourg, Madrid, Malmö, Manchester, Marseille, Melbourne, Miami FL, Milan, Montreal, New York NY, Osaka, Oslo, Paris, Perth, Phoenix AZ, Prague, Raleigh NC, Riga, Rome, Salt Lake City UT, San Jose CA, Sao Paulo, Seattle WA, Secaucus NJ, Singapore, Skopje, Sofia, Stavanger, Stockholm, Sydney, Tallinn, Tel Aviv, Tirana, Tokyo, Toronto, Vancouver, Vienna, Warsaw, Zagreb, Zurich
```

**Comment by Sembemann** [comment]
Same issue here.

**Comment by Leeharg** [comment]
Same issue as above when using SERVER_REGIONS=Netherlands and private internet access.

**Comment by atlastheshark** [comment]
Same issue. Fixed it by fixing the version to `qmcgaw/gluetun:v3.34.1` in my compose file. 
Running with Wireguard seems to have another issue where the health check fails, so switch over to OpenVPN for now.

**Comment by atticusmatticus** [comment]
The fix for @atlastheshark doesn't work for me. Still get same error message `ERROR VPN settings:...` Using Mullvad `SERVER_COUNTRIES=USA` (have tried with multiple countries and cities). Also have tried multiple versions vis-a-vis @atlastheshark's fix above with no luck.

**Comment by atlastheshark** [comment]
Here's my compose file:
```
version: "3"
services:
  gluetun:
    image: qmcgaw/gluetun:v3.34.1
    container_name: gluetun
    cap_add:
      - NET_ADMIN
    devices:
      - /dev/net/tun:/dev/net/tun
    ports:
      - 8888:8888/tcp # HTTP proxy
      - 8388:8388/tcp # Shadowsocks
      - 8388:8388/udp # Shadowsocks
      - 8080:8080 # qbitorrent
      - 6881:6881 # qbitorrent
      - 6881:6881/udp # qbitorrent
    volumes:
      - /[file path]/Config/gluetun:/gluetun
    environment:
      - VPN_SERVICE_PROVIDER=mullvad
      - VPN_TYPE=openvpn
      - OPENVPN_USER=[user ID]
      - SERVER_CITIES=Toronto

**Comment by martian67** [comment]
Same issue here.  Docker compose environment variable shows as all lower case and thus not matched a server location.

**Comment by roadkingvrod** [comment]
It works with v3.34 (not 3.34.1).

**Comment by SnippetSpace** [comment]
> 

can confirm

**Comment by MrBlueSea** [comment]
Also had the same issue and had to set it to v3.34.1 to fix this

**Comment by oester** [comment]
reverted to 3.34 to fix it for me (Mullvard, Toronto)

**Comment by Fastjur** [comment]
Just pointing out that this has been reported already :)

https://github.com/qdm12/gluetun/issues/1599

**Comment by qdm12** [comment]
The latest image `qmcgaw/gluetun` was fixed a few minutes ago by 7e325715c7a24da73447b29cec94b207bd950745.
The regression was introduced 2 days ago with a43973c093662c2b5521eb2e9e3aea053584e96b which did compare text taking care about case sensitivity (gluetun lowercases everything inputted, but keeps servers data text with case sensitivity). My apologies for the delay fixing it, obviously, and thank you for reporting the issue 🎖️ 

Just in case you are not aware, you can use released docker images such as `qmcgaw/gluetun:v3` or `qmcgaw/gluetun:v3.34.1` which should be more stable than the latest image `qmcgaw/gluetun`. I always appreciate users using the latest image since this helps fix bugs before bringing them to a stable release which would end up unstable, so ideally keep using the latest image 😉
